pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1204,6 +1179,45 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, allowEmul
 	return oldSpec, nil
 }
 
+func (l *LibvirtDomainManager) findOrCreateVirDomain(
```

Comment:
Can we use `lookup` instead of `find`?

Reply:
Yes, of course. Can you clarify the rationale so I can educate myself for future function nomenclature?

Reply:
Maybe it is a matter of taste, but I saw `lookup` referred to in technical terms more often than `find`.

The only difference I saw on the web is: https://wikidiff.com/find/lookup .